tests/src/Simple/ForegroundThreading/ForegroundThreading.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
https://dev.azure.com/dnceng/public/_build/results?buildId=854034&view=logs&j=945aca57-f06a-5a50-c259-721b2d8ad21f&t=5099bffa-6720-5a4a-47b3-c8a421944cb6&l=749
> tests/runtest.sh: line 30: /Users/runner/work/1/s/tests/src/Simple/ForegroundThreading/ForegroundThreading.sh: Permission denied

I guess I had to chmod it.